specification/synapse/resource-manager/Microsoft.Synapse/preview/2022-03-01-preview/bigDataPool.json

```diff
@@ -0,0 +1,670 @@
+            "type": "string"
+          },
+          "description": "Updated tags for the Big Data pool",
+          "type": "object"
+        }
+      }
+    },
+    "BigDataPoolResourceInfo": {
+      "type": "object",
+      "description": "A Big Data pool",
+      "title": "Big Data pool",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/BigDataPoolResourceProperties",
+          "description": "Big Data pool properties",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "BigDataPoolResourceProperties": {
+      "type": "object",
+      "description": "Properties of a Big Data pool powered by Apache Spark",
+      "title": "Spark pool properties",
+      "properties": {
+        "provisioningState": {
+          "type": "string",
+          "description": "The state of the Big Data pool."
+        },
+        "autoScale": {
+          "$ref": "#/definitions/AutoScaleProperties",
+          "description": "Auto-scaling properties"
+        },
+        "creationDate": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The time when the Big Data pool was created.",
+          "readOnly": true
+        },
+        "autoPause": {
+          "$ref": "#/definitions/AutoPauseProperties",
+          "description": "Auto-pausing properties"
+        },
+        "isComputeIsolationEnabled": {
+          "type": "boolean",
+          "description": "Whether compute isolation is required or not."
+        },
+        "isAutotuneEnabled": {
+          "type": "boolean",
+          "title": "Enable Autotune",
+          "description": "Whether autotune is required or not."
+        },
+        "sessionLevelPackagesEnabled": {
+          "type": "boolean",
+          "description": "Whether session level packages enabled."
+        },
+        "cacheSize": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The cache size"
+        },
+        "dynamicExecutorAllocation": {
+          "$ref": "#/definitions/DynamicExecutorAllocation",
+          "description": "Dynamic Executor Allocation"
+        },
+        "sparkEventsFolder": {
+          "type": "string",
+          "description": "The Spark events folder"
+        },
+        "nodeCount": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The number of nodes in the Big Data pool."
+        },
+        "libraryRequirements": {
+          "$ref": "#/definitions/LibraryRequirements",
+          "description": "Library version requirements"
+        },
+        "customLibraries": {
+          "description": "List of custom libraries/packages associated with the spark pool.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/LibraryInfo"
```

Comment:
Use this
            "$ref": "../../../../common/v1/types.json#/definitions/LibraryInfo"